macros/models/_sector/nft/nft_mints.sql

```diff
@@ -0,0 +1,189 @@
+{% macro nft_mints(blockchain, base_contracts, base_traces, erc20_transfer, base_transactions, eth_currency_contract ) %}
+
+WITH namespaces AS (
+    SELECT
+        address
+        , min_by(namespace, created_at) AS namespace
+	FROM {{ base_contracts }}
+	GROUP BY address
+	)
+
+, nfts_per_tx AS (
+    SELECT
+        tx_hash
+        , sum(amount) AS nfts_minted_in_tx
+    FROM {{ ref('nft_transfers') }}
+    WHERE
+        blockchain = '{{blockchain}}'
+        AND "from"= 0x0000000000000000000000000000000000000000
```

Comment:
think this line is impacting the differences in nft mint amount too as there are mints like [this](https://etherscan.io/tx/0x62c2f6f4a515c7ff3be033b833639fa0cfdac913323a14c7c58972a04a82217d) one where there are multiple transfers. if we only include from `0x0000000000000000000000000000000000000000`
then we are missing out other transfers 
cc: @0xRobin

Reply:
I don't think we want to double count this transfer though in your example?